cypress/integration/discountTest.js

```diff
@@ -0,0 +1,31 @@
+import HomePage from '../../src/pages/HomePage'
+import DiscountPage from '../../src/pages/DiscountPage'
+import * as urls from '../../testData/urls'
+
+describe("Verification of the discount sum", ()=>{
+    let priceOneValue = 0
+    let priceTwoValue = 0
+    let priceResultValue = 0
+    let testResultValue = 0
+
+    it('Verify sum of the discount', function() {
+        HomePage.openUrl(urls.salesUrl)
+        cy.get(DiscountPage.oneColumn).find('b').then(() => {     
+        Cypress.$(DiscountPage.oneColumn).each(function() {
+            priceOneValue += +Cypress.$(this).text()||0;
+            })
+        })
```

Comment:
Alternative implementation, which seems more easy for me:
```
        cy.get(DiscountPage.oneColumn).then(($col) => {
            var sum = 0;
            $col.each((index, el) => {
                cy.wrap(el).invoke('text').then(text => {
                    sum += parseFloat(text);
                })
            })
            cy.log('SUM').then(() => {
                ...... assertion
            })
        })
```

Reply:
I replaced my implementation with yours. Take a look at, please